contracts/meta-staking/src/testing/utils/mod.rs

```diff
@@ -0,0 +1,2 @@
+mod executes;
+mod queries;
```

Comment:
May as well make public.

```suggestion
pub mod executes;
pub mod queries;
```